gluten-core/src/main/resources/substrait/proto/substrait/algebra.proto

```diff
@@ -890,6 +892,17 @@ message Expression {
   }
 }
 
+message GenerateRel {
+  RelCommon common = 1;
+  Rel input = 2;
+  
+  Expression generator = 3;
```

Comment:
Should this relation be removed in favor of the official Substrait repo's ExpandRel?

Reply:
GenerateRel and ExpandRel is different. The former is used to represent spark `lateral view explode/posexpode`, and the latter is used to represent spark `grouping sets` or `with cube`